ekwipunek.py

```diff
@@ -0,0 +1,823 @@
+        for key, value in self.wyswietl_magazyn().items():
+            for k, v in value.items():
+                print(k + " sztuk " + str(len(v)))
+        val = input("Podaj nazwę przedmiotu, który chcesz wyrzucić")
+        if val == "wyjscie":
+            pass
+        else:
+            for key, value in self.wyswietl_magazyn().items():
+                if val in value:
+                    przedmiot = self.magazyn[key][val][0]
+                    Podloga_Obiekt.zawartosc.append(przedmiot)
+                    # mogę wyrzucić przedmiot w uzyciu, ale wtedy wylatuje on zarowno z w_uzyciu jak
+                    # i z kontenera (ZAWSZE), wynika to z linijki
+                    # przedmiot = self.magazyn[key][val][0]
+                    # poniewaz taka sama znajduje sie w metodzie uzyj (oraz Oselka)
+                    for k, v in self.w_uzyciu.copy().items():
+                        if przedmiot == v:
+                            del self.w_uzyciu[k]
+                    if isinstance(przedmiot, Bron):
+                        self._bronie[przedmiot.nazwa].remove(przedmiot)
+                        if len(self._bronie[przedmiot.nazwa]) == 0:
+                            self._bronie[przedmiot.nazwa].remove(przedmiot)
+                            del self._bronie[przedmiot.nazwa]
+                    elif isinstance(przedmiot, Pancerz):
+                        self._pancerze[przedmiot.nazwa].remove(przedmiot)
+                        if len(self._pancerze[przedmiot.nazwa]) == 0:
+                            del self._pancerze[przedmiot.nazwa]
+                    elif isinstance(przedmiot, Magia):
+                        self._przedmioty_magiczne[przedmiot.nazwa].remove(przedmiot)
+                        if len(self._przedmioty_magiczne[przedmiot.nazwa]) == 0:
+                            del self._przedmioty_magiczne[przedmiot.nazwa]
+                    elif isinstance(przedmiot, Pismo):
+                        self._pisma[przedmiot.nazwa].remove(przedmiot)
+                        if len(self._pisma[przedmiot.nazwa]) == 0:
+                            del self._pisma[przedmiot.nazwa]
+                    elif isinstance(przedmiot, Artefakt):
+                        self._artefakty[przedmiot.nazwa].remove(przedmiot)
+                        if len(self._artefakty[przedmiot.nazwa]) == 0:
+                            del self._artefakty[przedmiot.nazwa]
+                    elif isinstance(przedmiot, Jedzenie):
+                        self._jedzenie[przedmiot.nazwa].remove(przedmiot)
+                        if len(self._jedzenie[przedmiot.nazwa]) == 0:
+                            del self._jedzenie[przedmiot.nazwa]
+                    elif isinstance(przedmiot, Pozostale):
+                        self._pozostale[przedmiot.nazwa].remove(przedmiot)
+                        if len(self._pozostale[przedmiot.nazwa]) == 0:
+                            del self._pozostale[przedmiot.nazwa]
+            self.wyrzuc()
```

Comment:
A to czemu?

Reply:
aaa chyba wiem. lepiej użyć pętli while, która powinna być w metodzie otworz_ekwipunek

Reply:
właściwie to metoda otworz_ekwipunek powinna należeć do klasy Bohater

Reply:
Po klasie Postać powinna dziedziczyć klasa Bohater. O imieniu "Brak" xD

Reply:
Nie rozumiem dwóch pierwszych komentarzy :D

Reply:
### WHILE -> DOPOKI NIE WCISNE ESC TO BADZ SOBIE W TEJ FUNCKJI